www-client/netsurf/netsurf-3.8.recipe

```diff
@@ -24,6 +23,8 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
+	lib:libcss$secondaryArchSuffix
+	lib:libparserutils$secondaryArchSuffix
```

Comment:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
 	lib:libcss$secondaryArchSuffix
-	lib:libparserutils$secondaryArchSuffix
 	lib:libcurl$secondaryArchSuffix
+	lib:libdom$secondaryArchSuffix
 	lib:libexpat$secondaryArchSuffix
 	lib:libhubbub$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:libnsbmp$secondaryArchSuffix
 	lib:libnsgif$secondaryArchSuffix
+	lib:libparserutils$secondaryArchSuffix
 	lib:libpng16$secondaryArchSuffix
 	lib:libssl$secondaryArchSuffix
+	lib:libsvgtiny$secondaryArchSuffix
 	lib:libutf8proc$secondaryArchSuffix
 	lib:libwapcaplet$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
 	"
```
That's because `libdom` and `libsvgtiny` have been updated to also build their shared libs.

Reply:
Fixed